### DISCUSSION TITLE
How many detections is everyone getting?

### DISCUSSION THREAD

**Discussion by nbanderso** [mcguirepr89/BirdNET-Pi#854] [General]
Hello,

I've had birdnetpi running off and on for about a year now.  I'm curious what sort of data people are getting and how it compares to my part of the world.  For reference here's my info (based on today's detection tab:

- **Location**: Northern California (suburbs)

- **Time Active**: 10 months (approximate length of time BirdNET-pi has been actively running)

- **Total detections**: 80,905

- **Average daily detections**:  400 (very rough average based on the 'Today' column)

- **Unique Species Total**: 186

- **Average daily unique species** (very rough average based on the 'Unique Species Today' column):  22

**Comment by ehpersonal38** [comment]
- Location: Central Florida (more ruralish but still suburbs)
- Time Active: 363 days (almost a year exactly) 
- Total detections: 526,452
- Unique species total: 254
- Unique species daily: about 30-50
- Minimum confidence: 0.5
- Sigmoid sensitivity: 1.0

**Comment by Svardsten53** [comment]
**Location:** Tyresö, Sweden (outside of Stockholm)
**Time active:** 17 months
**Total detections:** 367,595
**Unique species total:** 141
**Minimum confidence:** 0.70
**Sigmoid sensitivity:** 1.25
**Model:** 3K
**BirdWeather ID:** 85
**Link:** https://birdnet.svardsten.se

**Reply by jmtuts** [reply]
Svardsten53 -  If you click on your Link you get a 404 error

**Reply by Svardsten53** [reply]
Thanks for pointing that out. The link is correct, but something went wrong in the github syntax. Now it works.

**Reply by jmtuts** [reply]
A question:  on your Oversikt page, the bar graphs have a total count of detections per species, how did you do that?

**Reply by blogdemoi** [reply]
I also think that showing weather data is quite a good idea! I would like to have this implemented in Birdnet-Pi... :-)

**Reply by Svardsten53** [reply]
I'm using an old version of daily_plot.py that I got help adapting to my wishes - to show all species and numbers for each species. I can't write code, but have tried pasting that snippet into the current code, but still no total count.

If you want to test it yourself, the code looks like this:

`# Function to show value on bars - from https://stackoverflow.com/questions/43214978/seaborn-barplot-displaying-values
def show_values_on_bars(ax,label):
    i = 0
    for p in ax.patches:
        _x = p.get_x() + p.get_width()* 0.9
        _y = p.get_y() + p.get_height() / 2
        value = '{:.0%}'.format(label[i])
        # Uncomment for Species Count Total
        value = '{:n}'.format(p.get_width())
        ax.text(_x, _y, value, ha='center', va='center', size=8, fontweight='bold', color='darkgreen', bbox=dict(facecolor='lightgrey',pad = 4.0))
        i=i+1`

**Reply by Svardsten53** [reply]
I get the weather data I use by using javascript against an API at the Swedish Meteorological and Hydrological Institute, where I download the last hour's observations from the nearest weather station. It is a local solution that unfortunately cannot be used in BirdNET-Pi for other countries.

For users in Sweden, I am happy to share the weather solution and also the translation and other adaptations I have made. Email address is on my page.

**Comment by hoover67** [comment]
Location: NW-Germany (Herford, BirdWeather station no.207) 
Running for about a year now (24/7)
Total detections 106772
Species: 92 (there a still afew dubious ones in there I am sure although I do my best trying to keeping things "clean" :-)). 
Min. confidence: 70
Sigmoid sens: 1.25

I recently updated my cheap 10€-Usb mic to a home-built AOM capsule based analog microphone conected to an external USB sound card, since then the daily number of detections has gone up significantly.

**Comment by jflyons** [comment]
Location: Intermountain USA, Boise, Idaho
Running for eight months, data since October
Total detections: 39,470 since October 6, 2022
Species: 123 (overdue for some serious clean-up)
Min. confidence: 70
Sigmoid sens: 1.0
Number of excluded species: 15

My average daily detections went up significantly with the new model, as it has proven much more adept at detecting my most commonly heard backyard birds - House Finch, Lesser Goldfinch and European Starling including the latter's dead-on Killdeer and other imitations.

**Comment by krummrey** [comment]
- Location: Ahrensburg, Germany (suburban garden on the outskirts of Hamburg)
- Time Active: 9 months
- Total detections: 244414
- Average daily detections: 2000-2500
- Unique Species Total: 167 (quite a few false positives in there.)
- Average daily unique species: 25-30
- Species Occurence Frequency Threshold: 0.01
- Minimum Confidence: 0.7
- Sigmoid Sensitivity:1

Detections went high up with the new model. Using a Pi3+ and a < €10,- USB-mic from amazon.
Birdweather ID 436

**Comment by lloydbayley** [comment]
Location: Berowra, New South Wales, Australia
Time active: 337 days (17/05/2022)
Total detections: 142082 (Recently had a crash so it's reading a bit less on the live site)
Unique species total: 83
Minimum confidence: 0.78
Sigmoid sensitivity: 1.25
BirdWeather ID: 200
Link: [https://birds.lloydsplace.au/](https://birds.lloydsplace.au/)

**Comment by jmtuts** [comment]
Location: Intermountain USA, Marana, Arizona
Running:  Feb 9, 2023, started with a RPI 3B+ a couple of months earlier but it had too much noise, so went to Libre Le Potato
Total detections: 58902
Species: 104
Avg Detections:  1,323 (Last 10 days, High = 1954, Low = 931)
Min. confidence: 0.7
Sigmoid sens: 1.25
Number of excluded species: 0
Site:  http://fallenbirdnet.sytes.net/

**Comment by blogdemoi** [comment]
Location: Garches, France
Running: Started about one year ago, with numerous new starts due to my poor Pi 3B, switched recently to a Libre Le Potato and everything is fine now.
Total detections: 42850
Species: 67 (a few bad detection)
Model: 3K
Min. confidence: 0.79
Sigmoid sens: 1.25
Number of excluded species: 1
Site: https://birdnetpi.arnauld.org/

**Comment by hugsworth** [comment]
Does anyone have suggestions for improving this? I am getting really really low results. Like a few birds a day. But when I listen in and when I watch the logs I am seeing high confidence birds that are not making it into detection and I am not sure why. I have tried two different condenser USB Microphones and they both provide very clear audio when I listen to the stream. Perhaps I am misunderstanding how the detection works. Is there any good documentation explaining what the text in the log is actually indicating? 

Location: Spokane, United States
Running: 3 days
![Screenshot 2023-04-29 105110](https://user-images.githubusercontent.com/76825501/235365297-8f450d11-32c9-4666-a7f2-91153093eb1f.png)

Total Detections: 10
Species:: 5
Model: 3K currently
Min Confidence: .77
Sigmoid sens: 1.30

**Reply by jflyons** [reply]
My understanding of the answer to your question is naive at best so I will stay tuned for any expert advice. I can relate.

With the 6K model I was having a hard time detecting House Finch up to a .7 level, despite being among my most common backyard birds. The 3K model seems to have little problem now (nor with Lesser Goldfinch, in the same category), though Song Sparrow and American Crow slip through undetected what seems like 95% of the time I am monitoring. 

Lots of interesting results and lots of questions!

**Comment by becausebirds** [comment]
Location: Central Texas (suburbs)
Time Active: 127 days
Total detections: 139,211
Unique species total: 129
Unique species daily: about 20-30
Average detections daily: >2,000
Minimum confidence: 0.5
Sigmoid sensitivity: 1.0

[Link to station](https://birdnet.becausebirds.com)
